### PR TITLE
Mariana huh

### PR DESCRIPTION
**Pull request by marianahuh**
@ielvisd. Completed MVP for JavaScript-IV.

### CODE REVIEW

**Comment by marianahuh** [comment]
Corrected a couple of functions parameters on listSubjects and debugsCode. @ielvisd

File: assignments/prototype-refactor.js [hunk 1]
```diff
@@ -7,3 +7,102 @@ Prototype Refactor
...
+    this.name = charaAttributes.name;
+  }
+  takeDamage() {
+    return (`${this.name} took damage.`);
+  }
+} //Child Class
+
+
+
+class Humanoid extends CharacterStats {
```
Comment (ielvisd) [inline assignments/prototype-refactor.js:34]: Great use of class constructor objects

File: assignments/prototype-refactor.js [hunk 2]
```diff
@@ -7,3 +7,102 @@ Prototype Refactor
...
+  constructor(gdAttributes) {
+    super(gdAttributes);
+    this.team = gdAttributes.team;
+    this.weapons = gdAttributes.weapons;
+    this.language = gdAttributes.language;
+  }
+  greet() {
+    return (`${this.name} offers a greeting in ${this.language}.`);
```
Comment (ielvisd) [inline assignments/prototype-refactor.js:42]: nice use of `this` and string template literals

**Review by ielvisd — APPROVED** [review]
Great work Mariana, writing some nice code.

**Pull request merged**